src/PhpcsDiff.php

```diff
@@ -50,11 +55,18 @@ public function __construct(array $argv, CLImate $climate)
         $this->argv = $argv;
         $this->climate = $climate;
 
-        if ($this->isFlagSet('-v')) {
+        if ($this->getFlag('-v')) {
             $this->climate->comment('Running in verbose mode.');
             $this->isVerbose = true;
         }
 
+        $standard = $this->getFlag('--standard');
+        if (!empty($standard) && is_string($standard)) {
+            $this->climate->comment('Custom standard: ' . $standard . PHP_EOL);
+
+            $this->standard = $standard;
```

Comment:
May be we should check that the file exists?

Reply:
@ecourtial I thought of adding the file check, but then something that crossed my mind is that a standard not always may be a file.

For example, if you use one of the standards that are included in phpcs ( https://github.com/squizlabs/PHP_CodeSniffer/tree/master/src/Standards ), there won't be a specific file for that.

Also, the phpcs itself does a check to confirm if the standard exists.

Thanks!